test/lux/eng-25448_deviated_type_scope/confd.conf

```diff
@@ -0,0 +1,322 @@
+<!-- -*- nxml -*- -->
```

Comment:
This file is probably not needed in the test.

Reply:
Damn, sorry, will fix.